content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,97 @@
-   shape, size or even color. It is also possible to add the maximum number of sits the table can
-   have.
+.. _floors_tables/frontend:
 
-.. _floors_tables/orders:
+From the POS frontend
+---------------------
 
-Register your table(s) orders
-=============================
+To create floors and tables from the frontend, :ref:`open a POS session <pos/session-start>`,
+click the hamburger menu icon :guilabel:`≡` on the upper right corner, and :guilabel:`Edit Plan` to
+enter the **edit mode**.
 
-To register an order, click on the respective table. By doing so, you are taken
-to your main interface.
+Add a floor by clicking :guilabel:`+ Add Floor`, which opens a pop-up window where you can choose a
+name for it.
 
-.. image:: floors_tables/restaurant_07.png
-    :align: center
-    :alt: View of the pos interface to register orders
+Once a floor is created, add a table by clicking :guilabel:`+ TABLE`. You can move it by selecting
+it and drag and dropping it as desired. Once a table is selected, you can also change the number of
+seats available by clicking :guilabel:`SEATS`, the table shape by clicking :guilabel:`SHAPE`, the
+table color by clicking :guilabel:`FILL`, or the table name by clicking :guilabel:`RENAME`. To
+duplicate an existing table, select it and click :guilabel:`COPY`. You can also remove a table by
+clicking :guilabel:`DELETE`.
 
-Transfer customer(s)
-====================
+Click :guilabel:`CLOSE` once you are satisfied with your floor plan to save it.
 
-If your customers want to move to another table after they already
-ordered, use the transfer button. This way, the order is also moved to the new
-table.
+.. image:: floors_tables/floor-map.png
 
-To do so, select the table your customer is currently on.
+.. note::
+   If no table is selected, the modification are applied to the floor.
 
-.. image:: floors_tables/restaurant_08.png
-    :align: center
-    :alt: View of the restaurant tables, one having a pending order
+.. warning::
+   Removing a table or a floor cannot be undone.
 
-Now, click on the transfer button and select the table to which you are transferring your
-customer.
+.. _floors_tables/orders:
 
-.. image:: floors_tables/restaurant_09.png
-    :align: center
-    :alt: View of the pos interface and transfer button. How to transfer customers from one table
-          to another
+Take orders
+===========
 
-Register an additional order
-============================
+Click a table to access the POS interface and start taking your customer's order. The system
+automatically associates the order to the chosen table, allowing you to add more items and generate
+a bill specifically tied to the orders placed at that table.
 
-When registering an order, use the + button to simultaneously proceed to
-another one.
+Once an order is placed, go back to the floor plan view by clicking :guilabel:`BACK` and select
+another table to register another order.
```

Comment:
Once an order is placed, return to the floor plan view by clicking :guilabel:`BACK` and selecting another table to register another order.